spec/javascript/orangelight/availability.spec.js

```diff
@@ -2,408 +2,411 @@ import updater from 'orangelight/availability';
-        expect(available_result.hasClass('badge')).toBe(true);
-        expect(available_result.hasClass('badge-success')).toBe(true);
-        expect(available_result.text()).toEqual("Available");
-
-        const unavailable_result = $(`*[data-record-id="${unavail_id}"] .availability-icon`);
-        expect(unavailable_result.hasClass('badge')).toBe(true);
-        expect(unavailable_result.hasClass('badge-danger')).toBe(true);
-        expect(unavailable_result.text()).toEqual("Unavailable");
-
-        const mixed_result = $(`*[data-record-id="${mixed_id}"] .availability-icon`);
-        expect(mixed_result.hasClass('badge')).toBe(true);
-        expect(mixed_result.hasClass('badge-secondary')).toBe(true);
-        expect(mixed_result.text()).toEqual("Some items not available");
-    });
-    test('search results - SCSB availability - Unavailable', () => {
-      document.body.innerHTML =
-      '<li class="blacklight-holdings">'+
-        '<ul>'+
-          '<li class="holding-status sm-checked" data-availability-record="false" data-record-id="SCSB-5467030" data-holding-id="5459517" data-aeon="false" data-bound-with="false">'+
-            '<span class="availability-icon badge badge-danger" data-scsb-availability="true" data-scsb-barcode="CU15957330">Not Available</span>'+
-            '<div class="library-location" data-location="true" data-record-id="SCSB-5467030" data-holding-id="5459517">'+
-              '<span class="results_location">ReCAP - Remote Storage</span> » <span class="call-number">PL5132 .B68 2007g</span>'+
-            '</div>'+
-          '</li>'+
-          '<li class="empty" data-record-id="SCSB-5467030">'+
-            '<a class="availability-icon more-info" href="/catalog/SCSB-5467030"></a>'+
-          '</li>'+
-        '</ul>'+
+    let bibdata_response = await fs.readFile("spec/fixtures/bibliographic_availability_3_bibs.json", 'utf8');
+    bibdata_response = JSON.parse(bibdata_response);
+
+    const u = new updater;
+    u.process_results_list(bibdata_response);
+
+    const available_result = $(`*[data-record-id="${avail_id}"] .availability-icon`);
+    expect(available_result.hasClass('badge')).toBe(true);
+    expect(available_result.hasClass('badge-success')).toBe(true);
+    expect(available_result.text()).toEqual("Available");
+
+    const unavailable_result = $(`*[data-record-id="${unavail_id}"] .availability-icon`);
+    expect(unavailable_result.hasClass('badge')).toBe(true);
+    expect(unavailable_result.hasClass('badge-danger')).toBe(true);
+    expect(unavailable_result.text()).toEqual("Unavailable");
+
+    const mixed_result = $(`*[data-record-id="${mixed_id}"] .availability-icon`);
+    expect(mixed_result.hasClass('badge')).toBe(true);
+    expect(mixed_result.hasClass('badge-secondary')).toBe(true);
+    expect(mixed_result.text()).toEqual("Some items not available");
+  });
+  test('search results - SCSB availability - Unavailable', () => {
```

Comment:
Updated spec for this PR. `test('search results - SCSB availability - Unavailable', () => {`

The rest are fixes from eslint/prettier.